xapian-core/include/xapian/termgenerator.h

```diff
@@ -141,6 +144,16 @@ class XAPIAN_VISIBILITY_DEFAULT TermGenerator {
      */
     void set_stemming_strategy(stem_strategy strategy);
 
+    /** Set the stopper strategy.
+     *
+     *  @param strategy The strategy to use - possible values are:
+     *   - STOP_NONE:     Don't ignore the word if it is a stop word.  
+     *   - STOP_IGNORE:   Ignore the word if it is a stop word. 
+     *   - STOP_INDEX_UNSTEMMED_ONLY: Ignore the stemmed form of the
+     *               word if it is a stop word.   
```

Comment:
I suspect this would be easier to understand if it was expressed the other way round. So instead of:

> Don't ignore the word if it is a stop word.

we could have:

> Index all words (don't use the stopper).

"Don't index" is probably better than 'Ignore' in the second, and the third should probably be something like "Only index the unstemmed…".

I wonder if the last constant should be named `STOP_STEMMED_FORMS` or something. And maybe the middle one could be `STOP_ALL` (which matches `STOP_NONE` better than `STOP_IGNORE`, I think). @ojwb do you have thoughts on these names?